arm/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/azure-cli:2.0.47
+
+LABEL version="1.0.0"
+
+LABEL maintainer="Disha Khakhar"
+LABEL com.github.actions.name="Manage Azure Resources"
+LABEL com.github.actions.description="GitHub Action to deploy ARM template to Azure"
+LABEL com.github.actions.icon="triange"
+LABEL com.github.actions.color="blue"
+
+RUN apk update \

```

Comment:
Why is this required?

Reply:
it is used to get the list of latest available packages, so this line is usually added just before apk add package

Reply:
I meant both the lines. Ok do we need util-linux? What for

Reply:
for generation of guid for deployment name. uuidgen is part of util-linux package